wp-includes/widgets/class-wp-widget-media.php

```diff
@@ -17,16 +17,32 @@
 abstract class WP_Widget_Media extends WP_Widget {
 
 	/**
-	 * Default instance.
+	 * Get instance schema.
 	 *
-	 * @todo The fields in this should be expanded out into full schema entries, with types and sanitize_callbacks.
-	 * @var array
+	 * This is protected because it may become part of WP_Widget eventually.
+	 *
+	 * @link https://core.trac.wordpress.org/ticket/35574
+	 * @return array
 	 */
-	protected $default_instance = array(
-		'attachment_id' => 0,
-		'url' => '',
-		'title' => '',
-	);
+	protected function get_instance_schema() {
```

Comment:
@obenland Yes, merging with the parent `get_instance_schema`. See the image widget. In that widget you can see that it's also making some calls in the schema definition.